### DISCUSSION TITLE
The Notifications count is not correct

### DISCUSSION THREAD

**Discussion by J-W-Chan** [community/community#49892] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The notifications list shows that I have unread notifications, but it is empty when I click in
![image](https://user-images.githubusercontent.com/6493028/224639327-bf90497b-903e-4904-9a2f-b5947ae8bbec.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @J-W-Chan ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by radar-blip** [comment]
Seems to be a ghost notification (known bug).
Here are some related posts that might help to solve the problem.

#6874

#48566

#35381

#35809
